ui/src/app/views/workspace-edit/component.html

```diff
@@ -8,7 +8,7 @@ <h2 *ngSwitchCase="Mode.Clone">Clone workspace "{{oldWorkspaceName}}"</h2>
   <div>
     <div class="name-area" [class.validation-error]="nameNotEntered">
       <input type="text" class="input name" name="workspace-name" autofocus
-          [(ngModel)]="workspace.name" [disabled]="!hasPermission"
+          [(ngModel)]="workspace.name" [disabled]="!hasPermission || awaitingReview"
```

Comment:
Hm, not sure I understand why we'd disable name edits while awaitingReview. Add comment?

Reply:
I think that, on thinking about it, this is totally fine to allow. Will remove.